docs/products/clickhouse/get-started.md

```diff
@@ -16,6 +16,19 @@ Start using Aiven for ClickHouse® by creating a service, adding a database, and
 
 ## Create a database
 
+Create a database either in the [Aiven Console](https://console.aiven.io/) or
+programmatically using the [Aiven API](/docs/tools/api).
+
+:::important
+Do not create a database using SQL.
```

Comment:
I'd say something like "you can not create a database using ClickHouse's SQL" because right now it seems like it is possible but not recommended. However, it is not possible at all!